warp/backend.go

```diff
@@ -248,3 +194,13 @@ func (b *backend) GetMessage(messageID ids.ID) (*avalancheWarp.UnsignedMessage,
 
 	return unsignedMessage, nil
 }
+
+func (b *backend) signMessage(unsignedMessage *avalancheWarp.UnsignedMessage) ([]byte, error) {
+	sig, err := b.warpSigner.Sign(unsignedMessage)
+	if err != nil {
+		return []byte{}, fmt.Errorf("failed to sign warp message: %w", err)
```

Comment:
style nit: should we switch to returning `nil` in error cases instead of []byte{}?